engines/group/app/controllers/group/api/v1/groups_controller.rb

```diff
@@ -81,19 +81,7 @@ def initialize_side_effects
         def add_side_effects_links
           return unless @side_effects.any?
 
-          links = @side_effects.flatten.map do |side_effect|
-            "<#{api_link(side_effect)}>; rel=\"created\" type=\"#{model_type(side_effect)}\""
-          end.join(', ')
-
-          response.headers['Link'] = links
-        end
-
-        def api_link(side_effect)
-          "http://localhost:8000/api/v1/#{model_type(side_effect).downcase.pluralize}/#{side_effect.id}"
-        end
-
-        def model_type(side_effect)
-          side_effect.class.name
+          response.headers['Link'] = ::Shared::HttpResourceLinks.build(@side_effects.flatten.uniq)
```

Comment:
@sauloperez can we make the following changes to hound?

 - 100 chars per line
 - prefer single quotes unless you need doubles...